ci-operator/config/openshift/release/openshift-release-master__nightly-4.11-upgrade-from-stable-4.10.yaml

```diff
@@ -79,6 +79,26 @@ tests:
       BASE_DOMAIN: aws-2.ci.openshift.org
       TEST_TYPE: upgrade-conformance
     workflow: openshift-upgrade-aws
+- as: e2e-aws-upgrade-single-node
+  interval: 24h
+  steps:
+    cluster_profile: aws-2
+    env:
+      BASE_DOMAIN: aws-2.ci.openshift.org
+      TEST_SKIPS: static pods should start after being created
+      TEST_TYPE: upgrade
```

Comment:
Is this temporary? It should be `upgrade-conformance`

Reply:
`upgrade-conformance` ignores `TEST_UPGRADE_SUITE: none` so we'll see disruption tests failing

Reply:
The disruption tests are failing because https://github.com/openshift/origin/pull/26929 didn't merge yet, but we're supposed to run them